src/espIdf/debugAdapter/debugAdapterManager.ts

```diff
@@ -117,6 +132,8 @@ export class DebugAdapterManager extends EventEmitter {
         this.port.toString(),
         "-dn",
         this.target,
+        "-a",
+        model.app.address,
```

Comment:
I got a little further by changing this to:
`parseInt(model.app.address, 16).toString(),`
but Gdb is still using 0x10000 as shown in the debug log

Reply:
> Here is the log showing GDB still running `mon esp appimage_offset 0x10000` command:
> [debug.log](https://github.com/espressif/vscode-esp-idf-extension/files/5650117/debug.log)
> EDIT:
> I found a workaround for the GDB issue. In launch.json, I set:
> 
> ```
> {
>             "type": "espidf",
>             "name": "Launch",
>             "request": "launch",
>             "logLevel": 5,
>             "gdbinitFile": "${workspaceFolder}/build/gdbinit",
> 
>         },
> ```
> 
> gdbinit:
> 
> ```
> target remote :3333
> symbol-file /home/neal/esp/esp-rainmaker/examples/led_light/build/led_light.elf
> mon reset halt
> flushregs
> thb app_main
> c
> ```
> 
> I can then debug without error. I still have that line edited though with `parseInt`

If `gdbinit` file is not specified, then https://github.com/espressif/esp-debug-adapter/blob/5439cd3c6e5632d8fef6754b59d2d3820f335854/debug_adapter/debug_backend/debug_backend/hw_specific/esp_init.gdb is used. Need to look into it.

Reply:
Can you share the elf file ?

Reply:
Still want the led_light.elf file?